docs_new/blog/index.md

```diff
@@ -0,0 +1 @@
+# Blog
```

Comment:
This should be deleted. It keeps appearing for some reason, I've deleted it a few times. Not sure where that's coming from.